frr/zebra_dplane_grout.c

```diff
@@ -638,8 +643,15 @@ static int zd_grout_process(struct zebra_dplane_provider *prov) {
 static void zd_grout_ns(struct event *t) {
 	struct event_loop *dg_master = dplane_get_thread_master();
 	struct vrf *default_vrf;
+	struct ns *default_ns;
 
-	zebra_ns_disabled(ns_get_default());
+	default_ns = ns_get_default();
+	if (!default_ns) {
+		gr_log_err("no default namespace");
+		exit(1);
+	}
+	netlink_add_dummy_link(default_ns->info, DEFAULT_SRV6_IFNAME, GROUT_SRV6_IFINDEX);
+	zebra_ns_disabled(default_ns);
```

Comment:
This seems to contradict what you did in the previous commit. I thought the fake sr0 interface in zebra was enough. What is the point in creating a real dummy interface in linux?